rss-client/src/app/app-inventory/components/user-inventory-item/user-inventory-item.component.ts

```diff
@@ -0,0 +1,208 @@
+import { Component, OnInit, Input } from '@angular/core';
```

Comment:
Is this new code or was this ts file just moved from somewhere else? If it is the later just let me know here and you can ignore the comments below since you inherited the issues.

Reply:
The inventory-item component was duplicated into the user-inventory-item component to allow for modularity: now, when an Employee user clicks the detail button, this modal will pop up. Much of the code was copy-and-pasted from the inventory-item component, and much of it was left alone because when we tried untangling it, things broke in bad ways.